tests/cql/CqlListOperatorsTest.xml

```diff
@@ -112,11 +132,11 @@
 			<output>null</output>
 		</test>
 		<test name="EqualEmptyListNull">
-			<expression>{} = null</expression>
+			<expression>{} as List&lt;String&gt; = null</expression>
```

Comment:
More fun w/ casting here and on line 139 to keep the translator happy when `null` could be one of many types.